.travis.yml

```diff
@@ -1,20 +1,25 @@
 language: go
 
 go:
-  - 1.8
-  - tip
-
-matrix:
-  fast_finish: true
-  allow_failures:
-    - go: tip
+  - 1.9
```

Comment:
Maybe we should continue testing on Go 1.8 (and 1.9), it's still the default version for Ubuntu and other distros.

Reply:
This is a compiled program, we compile for 1.9 and the binaries and docker are release to with version.